default.nix

```diff
@@ -1,12 +1,9 @@
-attrs@{ ... }:
-
-let
-  localPackages = import ./nix/localPackages.nix attrs;
-in
-{
-  inherit (localPackages)
-    rhine
-    rhine-gloss
-    rhine-examples
-  ;
-}
+(import (
```

Comment:
I think there is a way to detect in a `default.nix` whether it is called from `nix build` or `nix shell`. This could be used to remove `shell.nix` completely. But I can't find the option.

Reply:
If we're ok with requiring a recent version of nix (from `22.05` it's the default, and before it requires setting `experimental-features = nix-command flakes`, then we actually need neither `default.nix` nor `shell.nix` (which are only here to [support legacy nix](https://nixos.wiki/wiki/Flakes#Using_flakes_project_from_a_legacy_Nix)).

Reply:
Yes, that's also ok. Let's drop support for legacy nix.

Reply:
Or rather, I'll leave the choice up to you. Whatever you're more comfortable with.